src/Page.js

```diff
@@ -849,6 +852,22 @@ Page.prototype.postRender = function (content) {
   return postRenderedContent;
 };
 
+/**
+ * Collect page content inserted by plugins
+ */
+Page.prototype.collectPluginsContent = function () {
+  let links = [];
+  let scripts = [];
+
+  Object.entries(this.plugins).forEach(([pluginName, plugin]) => {
+    if (plugin.addLinks) {
+      links = links.concat(plugin.addLinks(this.pluginsContext[pluginName], this.frontMatter));
+      scripts = scripts.concat(plugin.addScripts(this.pluginsContext[pluginName], this.frontMatter));
```

Comment:
- Shall we pass in `content` as the first parameter for consistency?
- Shall we pass in `utils` as the fourth parameter, with methods like `buildScript(src)`?

Reply:
What would those methods do?

Reply:
For now, something simple like <code>(src) =>  \`<script src="${src}"></script>`</code>.

Usage: `utils.buildScript('SCRIPT_LINK')`